Remotion/Core/UnitTests/Context/SafeContextIntegrationTest.cs

```diff
@@ -24,7 +24,7 @@ namespace Remotion.UnitTests.Context
   [TestFixture]
   public class SafeContextIntegrationTest
   {
-    [Test]
+    [Test, Ignore ("Testing via CallContext is no longer possible.")]
```

Comment:
@MichaelKetting what should happen to those tests when asserting via CallContext queries is no longer possible?

Reply:
@iam-mholle The basic test logic can remain (unless it's redundant with the unit tests?). As for the rest, it would be helpful if we could set up threaded and async tests to show that a) there's no cross talk between new threads and b) correct behavior exists for async calls. If necesarry, we can split this and only do the threaded tests for now to show the values are isolated between threadpool-threads.